debian/helpers/gitcommit.sh

```diff
@@ -5,7 +5,7 @@ uVersion="$1"
 dVersion="$2"
 
 if [ -z "$uVersion" ]; then
-	uVersion="$(< VERSION)"
+	uVersion="$(< cli/VERSION)"
```

Comment:
Was this actually accurate?  (I thought you'd mentioned it was still `-dev`?)

Reply:
Yes, it contains `-dev` indeed. Initially, I just commented out this `if` statement but I thought it'd not be the best thing to do. This script is also used by the integration tests. I tried to not change too much the current implementation, but if you have a suggestion of a solution I'll be happy to implement it.

Reply:
Yeah, I think the `-dev` suffix is killing the ability to know this is a release, so it's doing `git rev-parse` which is going to fail on the actual builders and thus likely fail to build.

I think our best course of action now is to delete this script and set `DOCKER_GITCOMMIT` to something that identifies the origin of the build (which is the intent of this field anyhow), such as the full Ubuntu version number.

Reply:
what exactly would be the "full Ubuntu version number"? Is there a way to get directly from a variable? Like we did for `DEB_VERSION_UPSTREAM`.

Reply:
I think `DEB_VERSION` is probably what I'd propose.

Reply:
I pushed a new commit trying to achieve this, please take a look.